### PR TITLE
Remove startup tasks from DTD

### PR DESCRIPTION
**Pull request by therault**
This is the only commits from the 'various' set of commits in parsec-ttg that have not been merged into master yet.

I believe there is another branch/PR with a larger set of cleanup for DTD, so maybe we just want to drop this.

### CODE REVIEW

**Comment by bosilca** [comment]
This is indeed a part among the goodies of #323, but we can merge it as a standalone effort, and I will update my branch.

**Review by bosilca — APPROVED** [review]

File: parsec/mca/termdet/user_trigger/termdet_user_trigger_module.c [hunk 1]
```diff
@@ -156,7 +156,7 @@ static void parsec_termdet_user_trigger_monitor_taskpool(parsec_taskpool_t *tp,
 {
     parsec_termdet_user_trigger_monitor_t *monitor;
     assert(&parsec_termdet_user_trigger_module.module == tp->tdm.module);
-    assert(NULL == tp->tdm.monitor);
```
Comment (abouteiller) [inline parsec/mca/termdet/user_trigger/termdet_user_trigger_module.c:159]: should this be reinstated or removed?

Comment (therault) [reply parsec/mca/termdet/user_trigger/termdet_user_trigger_module.c:159]: The patch says to remove it

Comment (abouteiller) [reply parsec/mca/termdet/user_trigger/termdet_user_trigger_module.c:159]: no the patch comments it line below

**Pull request merged**